### ISSUE TITLE
NullPointerException in PhoneNumberUtils

### ISSUE THREAD

**Issue by krawa** [firebase/FirebaseUI-Android#780]
Android 4.4
Samsung Galaxy Ace 4 (vivaltods5m)

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.deliverResults(ActivityThread.java:3710)
  at android.app.ActivityThread.handleSendResult(ActivityThread.java:3753)
  at android.app.ActivityThread.access$1400(ActivityThread.java:179)
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1372)
  at android.os.Handler.dispatchMessage(Handler.java:102)
  at android.os.Looper.loop(Looper.java:146)
  at android.app.ActivityThread.main(ActivityThread.java:5641)
  at java.lang.reflect.Method.invokeNative(Method.java:0)
  at java.lang.reflect.Method.invoke(Method.java:515)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1288)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1104)
  at dalvik.system.NativeStart.main(NativeStart.java:0)
Caused by: java.lang.NullPointerException: 
  at com.firebase.ui.auth.ui.phone.PhoneNumberUtils.getSimBasedLocale(PhoneNumberUtils.java:1342)
  at com.firebase.ui.auth.ui.phone.PhoneNumberUtils.getCurrentCountryInfo(PhoneNumberUtils.java:105)
  at com.firebase.ui.auth.ui.phone.PhoneNumberUtils.formatPhoneNumberUsingCurrentCountry(PhoneNumberUtils.java:99)
  at com.firebase.ui.auth.ui.phone.VerifyPhoneNumberFragment.onActivityResult(VerifyPhoneNumberFragment.java:144)
  at android.support.v4.app.FragmentActivity.onActivityResult(FragmentActivity.java:164)
  at android.app.Activity.dispatchActivityResult(Activity.java:5676)
  at android.app.ActivityThread.deliverResults(ActivityThread.java:3706)
```

**Comment by samtstern** [comment]
The method in question is:

```java
    private static Locale getSimBasedLocale(@NonNull Context context) {
        final TelephonyManager tm =
                (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
        final String countryIso = tm != null ? tm.getSimCountryIso() : null;
        return TextUtils.isEmpty(countryIso) ? null : new Locale("", countryIso);
    }
```

Line `1342` is `context.getSystemService(....)` so maybe we are passing a `null` Context into this method from somewhere?

**Comment by samtstern** [comment]
The fix for this issue has been released in `v2.1.0`.
